src/platforms/common/configuration/options.mdx

```diff
@@ -242,8 +242,13 @@ If possible, we recommended turning on this feature to send all such data by def
 
 <ConfigKey name="server-name" supported={["python", "node", "ruby", "php", "java", "dart", "dotnet"]} notSupported={["android"]}>
 
-This option can be used to supply a "server name." When provided, the name of the server is sent along and persisted in the event. For many integrations the server name actually corresponds to the device hostname, even in situations where the machine is not actually a server. Most SDKs
-will attempt to auto-discover this value.
+This option can be used to supply a "server name." When provided, the name of the server is sent along and persisted in the event. For many integrations the server name actually corresponds to the device hostname, even in situations where the machine is not actually a server.
+
+<PlatformSection notSupported={["dotnet"]}>
```

Comment:
.NET also does this but hides behind `SendDefaultPii` if not on ASPNET (when we know for sure it's a server).

The reason is taht it could be a desktop app, so that means we'd be sending a device name, potentiall "Bruno's Computer". That's why the opt-in through htat PII guard

Reply:
Can you suggest an edit @bruno-garcia?

Reply:
I pushed an update that explains this better.